### DISCUSSION TITLE
some alerts not working

### DISCUSSION THREAD

**Discussion by jhautot** [NightscoutFoundation/xDrip#2904] [Q&A]
Hello,

I try to set up some alerts without success.
If I unable regular low or high alerts it does work but I'm not interested in those ones because they already exist on the pump.
So I tried 2 different alerts that would be very interesting:
- low prediction
- fast descent and fast ascent. (I tried 1 and 3 as a value)

Both of them never works on my side.

I think I have all permissions enabled.

Do you have any idea on the reason why it does not work?

I'm using the latest version 

Best regards,

Jordi.

**Comment by Navid200** [comment]
Post screenshots of the forecast low alert and other alerts pages so that we can see your settings.

**Comment by jhautot** [comment]
![Screenshot_20230625_085452_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/24245993/4eac2437-f5a3-4815-a8be-64ac4019d27b)
![Screenshot_20230625_085448_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/24245993/e5033ed4-ce95-44d0-b242-37465de534f5)
![Screenshot_20230625_085432_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/24245993/87eccc3e-d75a-4fb4-a387-b7479aa789bc)
![Screenshot_20230625_085102_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/24245993/005f4a05-6c50-48d3-92e0-84fcfd112ea1)
![Screenshot_20230625_085131_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/24245993/c633a4db-e3a1-4739-9670-82ca64d7d8be)

**Reply by Navid200** [reply]
Can you check the xDrip logs?
Do you see the alerts you are interested in in the logs when you expected them?

If not, please note that the rapid rise and fall alerts need more than just two consecutive readings at greater than the target trajectory in order to trigger.

If you do see the logs but don't hear them, please note that the volume for those alerts is controlled by the ring and notification volume of your phone.
Is it possible that you have lowered that particular volume too much?

**Reply by jhautot** [reply]
Indeed It was working, I'm very sorry for your loss of time.
I didn't expect that notifications were muted on my wife's phone. 
In addition there are some tests of alarm sounds (in xdrip menu) that was working so I never imagine it could be that.

And I didn't know about this logs feature before. So thank you for pointing this to me.

Thanks a lot for your help!

Jordi.

**Reply by Navid200** [reply]
You're welcome

**Comment by jhautot** [comment]
Another question please,
Is it possible to setup working hours for those kinds of alerts (low prediction and big drop/rise).
Typical issue: lots of low prediction during night but I don't want any alert because the pump can deal with that. The blood sugar fluctuates between 100mg/dl and 70 all over the night due to the target and auto-adjustment of the insulin.

**Reply by Navid200** [reply]
No, those alerts don't have time of day.
I have an open issue that list the existing inconsistencies in the alerts:
https://github.com/NightscoutFoundation/xDrip/issues/1762

It is my goal to fix all of those.  However, that is a very challenging change.  It will take me a while.

**Comment by jhautot** [comment]
Thank you for your answer!
